server/migrations/20230726210735-create-skills.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Skills', {
+      skill_id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      skill_name: {
+        type: Sequelize.STRING,
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+    });
+  },
+  async down(queryInterface) {
+    await queryInterface.dropTable('Skills');
+  },
```

Comment:
it's preferable to have all migrations in the same migration (be sure to wrap it in a transaction)

Reply:
and as a general note, all tables should be snake_cased

Reply:
`createdAt` and `updatedAt` should also be snake case, as seen in the [flags table migration](https://github.com/timmyichen/dev-directory/blob/8931e3eecfc81a3b591ba73f0a0b607a461ece1e/server/migrations/20230611233941-create-flags-table.js#L27).

Reply:
> it's preferable to have all migrations in the same migration (be sure to wrap it in a transaction)

do you mean consolidating all the tables migrations into one? not 100% sure what you mean

Reply:
yes - one migration file